packages/frontend-2/components/billing/Alert.vue

```diff
@@ -1,11 +1,14 @@
+<!-- eslint-disable vuejs-accessibility/heading-has-content -->
```

Comment:
There is actually content in the titel, but because it's in a nested component linter doesnt see it